databroker/broker.py

```diff
@@ -260,9 +260,12 @@ def __init__(self, mds, fs, plugins=None, filters=None):
         filters : list
             list of mongo queries to be combined with query using '$and',
             acting as a filter to restrict the results
+        es : EventStoreRO, optional
+            If not provides, mds is expected to provide event storage.
         """
         self.mds = mds
         self.fs = fs
+        self.es = es
```

Comment:
I suggest doing this by assigning es as mds if es is None and making it required in `core`